### ISSUE TITLE
Reproduce results

### ISSUE THREAD

**Issue by AnaRhisT94** [LetianLee/MSc-Project-SemEval2020-Task4#1]
Hi,
I'm trying to reproduce results on google colab, I changed the `batch_size = 8` in order to use `roberta-large` model and the results stay the same (0.5 accuracy). Any ideas why?

![image](https://user-images.githubusercontent.com/33227900/154847823-a1b9d459-259c-44d6-aa3d-5a2efb551a03.png)

**Comment by AnaRhisT94** [comment]
Looks like using this mode gives good results:
```
model_name = "bert-base-uncased"
optimizer_name = 'Adam'
learning_rate = 1e-5
batch_size = 32
epoch = 10
```

I believe that 8 batches doesn't help the network to generalize well enough after backprop.
![image](https://user-images.githubusercontent.com/33227900/154852560-d06095b7-0122-48d2-af01-e66489e08e71.png)

**Comment by LetianLee** [comment]
Hi there!
Glad someone views my code. You are always welcome!

For this question, I did a huge number of experiments to study the effects of different hyperparameters.
You can try to see my experiment result in the file: 
[**MSc-Project-SemEval2020-Task4/MSc Project/Experiment/Experiment Workflow.ipynb**](https://github.com/LetianLee/MSc-Project-SemEval2020-Task4/blob/master/MSc%20Project/Experiment/Experiment%20Workflow.ipynb)

Thanks.

**Comment by LetianLee** [comment]
Hi there!

Glad someone views my code. You are always welcome!

For this question, I did a huge number of experiments to study the effects
of different hyperparameters.
You can try to see my experiment result in the file:
MSc-Project-SemEval2020-Task4/MSc Project/Experiment/Experiment
Workflow.ipynb
<https://github.com/LetianLee/MSc-Project-SemEval2020-Task4/blob/master/MSc%20Project/Experiment/Experiment%20Workflow.ipynb>

Thanks.



Ilan Aizelman ***@***.***> 于2022年2月20日周日 14:34写道：

**Comment by AnaRhisT94** [comment]
I rewrote some of the code to classify if a sentence is common sense or not from the comVE dataset and I reproduced ur results and got around 0.89-0.90% accuracy on test set.

Thanks a lot for ur work!

Any ideas where I can find a dataset with 100k+ annotations of commonsense validation? Instead of manually creating..
